site/app/libraries/socket/Server.php

```diff
@@ -107,9 +107,9 @@ private function checkAuth(ConnectionInterface $conn): bool {
     /**
      * Push a given message to all-but-sender connections on the same course and page
      */
-    private function broadcast(ConnectionInterface $from, string $content, string $page_name): void {
```

Comment:
Please revert all changes to this file. This should be done in a separate PR once all client side socket calls are removed.